providers/notificationProvider.tsx

```diff
@@ -0,0 +1,15 @@
+import * as Toast from '@radix-ui/react-toast';
+import type { CSSProperties, FC, PropsWithChildren } from 'react';
+
+const viewportStyles: CSSProperties = {
```

Comment:
I guess here we can use `satisfies` as it technically it is not the type of `CSSPRoperties` but satisfies it